### DISCUSSION TITLE
How can i create my own pretrained model

### DISCUSSION THREAD

**Discussion by Prayag22** [facebookresearch/detectron2#5172] [Ideas]
I have a Detectron2 pre-trained model resnet50.pkl that was originally trained on the COCO 2017 dataset. Now, I want to make it a custom pre-trained model by further training it on my own dataset.What i did till now is i trained as a proof of concept trained model on colab on balloon dataset exported model .pth format convert it to .pkl using convert-torchvision-to-d2.py however i got an error so replace the lines `newmodel[k] = obj.pop(old_k).detach().numpy() with newmodel[k] = obj.pop(old_k)` however it could not train a model using this. Any advice or suggestions would be highly appreciated!

**Comment by shruts37** [comment]
Figured out a way yet?

**Reply by Prayag22** [reply]
Not yet @shruts37
